plugins/include/message_const.inc

```diff
@@ -1151,6 +1151,25 @@ enum BreakModelFlags
  */
 #define TE_MULTIGUNSHOT             126
 
+/**
+ * Colors used in te_create_user_tracer()
+ */
+enum TracerColors
+{
+	TracerColor_White = 0,
+	TracerColor_Red,
+	TracerColor_Green,
+	TracerColor_Blue,
+	TracerColor_Yellow,
```

Comment:
Isn't it black?

Reply:
@WPMGPRoSToTeMa Nope, unless it's uncolored - https://i.imgur.com/1rqQ1r7.jpg
But the 12th color definitely isn't black as well - https://i.imgur.com/3rbtdlx.jpg

Reply:
@OciXCrom yeah, 12th color is buggy and invalid, it'll be `000080` on Windows client and `000000` on Linux client (what about Mac? lol). 4th color is based on `tracerred`, `tracergreen`, `tracerblue` and `traceralpha` cvars.

Reply:
@WPMGPRoSToTeMa Thanks for this information, I updated the documentation.